wiki/fr/fr-Onboarding.md

```diff
@@ -0,0 +1,58 @@
+---
+layout: wiki
+title: "Embarquement Jamulus"
+lang: "fr"
+permalink: "/wiki/Onboarding"
+---
+# Embarquement Jamulus
+Voyons à quoi Jamulus ressemble et ce que vous devez faire après avoir installé le logiciel.
+## Premier démarrage
+Avant de démarrer Jamulus 
+1. Pour l'instant, **fermez toutes les autres applications**. Il est préférable de commencer avec aucune au début.
+2. **Branchez tout** (câble Ethernet, matériel audio, etc.) 
+
+… pour *réduire la charge* sur votre ordinateur et vous assurez qu'*aucune autre application* essaie d'utiliser votre carte son.
+
+Vous pouvez jeter un coup d'œil à la [configuration matérielle](Hardware-Setup) pour plus d'informations sur ce que vous devez faire avec votre équipement. Si vous avez besoin d'aide pour configurer ASIO4All sur Windows, consultez le [guide de configuration ASIO4All sur la page installation sous Windows](Installation-for-Windows#configuration-de-asio4all).
+
+### Fenêtre principale de Jamulus
+Quand vous lancerez Jamulus, vous allez voir une fenêtre qui ressemble à ceci :
+
+![Fenêtre principale de Jamulus](https://user-images.githubusercontent.com/9108457/101895743-384c4880-3ba8-11eb-80ef-e8e03a0edf86.png)
+
+### 1. Configurez un profil
+Pour que les autres sachent qui vous êtes, vous devez configurer votre *profil*. Pour le faire, cliquez sur « Vue » (barre des menus) > « Mon profil… ».  
+Maintenant vous allez voir :
+
+![Profil de musicien](https://user-images.githubusercontent.com/9108457/101895791-48fcbe80-3ba8-11eb-913e-d0e4e7d60666.png)
+
+Remplissez au minimum votre « Pseudo/nom » pour renseigner votre nom et fermez la fenêtre. 
+
+### 2. Connectez vous à un serveur
+Maintenant que vous êtes configuré, vous pouvez utiliser le bouton « Se connecter » dans la fenêtre principale de Jamulus pour rejoindre un serveur. Une fenêtre s'ouvre alors dans laquelle vous voyez les *serveurs regroupés par genre*. Pour choisir un autre genre, sélectionnez-en un dans la liste en haut à gauche. Après avoir sélectionné un genre et un serveur, rejoignez le serveur en cliquant sur « Se connecter ». Si vous rencontrez des problèmes ici, ils sont très probablement dus à un équipement audio mal configuré ou à des problèmes de réseau. Consultez la [page de dépannage du client](Client-Troubleshooting).
+
```

Comment:
Do you need a full stop here: `de réseau. Consultez` in french? Could we maybe make it one sentence (If that isn't the case in the english translation, we should fix it there too.)

Reply:
In the EN version:
_[…]wrongly configured audio gear or network issues. Have a look at the [Troubleshooting page](Client-Troubleshooting)._

We can use the both, it is understandable. I don't have a fixed opinion on this point.

Reply:
Yeah. Ok. For me it sounded a bit strange but not sure what @gilgongo thinks about this (especially for the english version)